content/environment-setup/git/_index.md

```diff
@@ -46,6 +49,8 @@ Note:
 
 - If any of these commands ask you if you want to continue, then type in `Y` then enter.
 - At some point, Termux will ask you for a GitHub password. When you type in a password then it looks like nothing is being typed. This is normal. Just type out your password like you normally would and press enter.
+- If you get the error `remote: Support for password authentication was removed on August 13, 2021.`, then you need to [generate an SSH key](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and [add it to your GitHub account](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account). Because as As of August 13, 2021, GitHub and other Git hosting platforms have removed support for password authentication when interacting with remote repositories. This change is part of an effort to enhance security and encourage the use of more secure authentication methods, that key can be used as a password replacement for command-line and API authentication. You can read more about this [https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/](https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/).
```

Comment:
Please don't start a sentence with `Because`. Maybe say: `"This is because ..."`